dash_auth/public_routes.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+import inspect
+import os
+
+from dash import Dash, callback
+from dash._callback import GLOBAL_CALLBACK_MAP
+from dash import get_app
+from flask import Flask
+from werkzeug.datastructures import ImmutableDict
+from werkzeug.routing import Map, MapAdapter, Rule
+
+# Add PUBLIC_ROUTES in the default Flask config
+DASH_PUBLIC_ASSETS_EXTENSIONS = "js,css,svg,png,webp,gif"
```

Comment:
@RenaudLN , I am not sure we should make assets public. It would be reasonable to me that routes for selected assets (or all assets) have to be whitelisted manually

Reply:
Asking people to whitelist their custom js scripts and stylesheets seems overkill to me. Not 100% sure about images so maybe I could drop those.